tests/test_cases/fchdir.c

```diff
@@ -0,0 +1,60 @@
+#undef _GNU_SOURCE
+#define _GNU_SOURCE
+
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#define MAX_PATH 1024
+
+int main(void) {
+    char path[MAX_PATH];
+    
+    // Get the current working directory
+    char* result = getcwd(path, sizeof(path));
+    if (result == NULL) {
+        perror("getcwd() error");
+        return EXIT_FAILURE;
+    }
+    
+    printf("current working directory is: %s :: %s\n", path, result);
+    fflush(stdout);
+    
+
+    // Open a directory
+    int fd = open("automated_tests/", O_RDWR);
```

Comment:
Just make sure to append `automated_tests/` to the current working directory path i.e. `result` variable and then open the directory.